requirements-dev.txt

```diff
@@ -1,11 +1,7 @@
-elasticsearch==6.1.1
-elasticsearch_dsl==6.1.0
-pyyaml==3.12
-pyjq==2.1.0
-ansicolors==1.1.8
+-r requirements.txt
```

Comment:
Oooh! I didn't know I could do that! Thank you!

Reply:
Woah, me neither 👍